index.html

```diff
@@ -266,137 +289,324 @@ <h4 class="packcage-title">Package Summery Details</h4>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Half Board</span>
+                  <span class="hightlight-text">Half Board</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">All Inclusive</span>
+                  <span class="hightlight-text">All Inclusive</span>
                 </li>
               </ul>
             </div>
             <div class="col">
-              <ul class="details-box__list">
-                <li class="details-box__title">Entertainment</li>
+              <ul class="details-box-list">
+                <li class="details-box-title">Entertainment</li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Live music</span>
+                  <span class="hightlight-text">Live music</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">DJ & Mauritian Dance Show</span>
+                  <span class="hightlight-text">DJ & Mauritian Dance Show</span>
                 </li>
               </ul>
             </div>
             <div class="col">
-              <ul class="details-box__list">
-                <li class="details-box__title">Check In & Check Out</li>
+              <ul class="details-box-list">
+                <li class="details-box-title">Check In & Check Out</li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Check In Time 14:00</span>
+                  <span class="hightlight-text">Check In Time 14:00</span>
                 </li>
                 <li>
                   <span class="icon icon-chekin check"></span>
-                  <span class="hightlight-list__text">Check Out Time 11:00</span>
+                  <span class="hightlight-text">Check Out Time 11:00</span>
                 </li>
               </ul>
             </div>
           </div>
-
         </div>
     </section>
+    <section class="other_section">
+      <div class="container">
+        <h2 class="title-h2">Other Packages</h2>
+        <div class="row">
+          <div class="col-md-8">
+            <div class="hotel-card">
+              <picture class="hotel-card__img">
+                <source srcset="./images/pk_1.webp 1x, 
+                ./images/pk_1_2x.webp 2x" type="image/webp" class="hotel-card__img">
+                <source srcset="./images/pk_1.jpg 1x, 
+                ./images/pk_1_2x.jpg 2x" type="image/jpeg" class="hotel-card__img">
+                <img src="./images/pk_1.jpg" alt="hotel blue heaven" class="hotel-card__img">
+              </picture>
```

Comment:
The same classes for picture and img. It is not good idea.
Why do you need classes for source tags? It doesn't work. Please, remove.

Reply:
> The same classes for picture and img. It is not good idea.
> Why do you need classes for source tags? It doesn't work. Please, remove.

because i use border radius for all types of pictures, and this class dont work for <picture>